features/credit_card_url.feature

```diff
@@ -30,3 +30,12 @@ Feature: Credit Card Form URL
 
     When I visit my profile page
     Then the new value should be stored as my Credit Card Form URL
+
+  Scenario: Vendor cannot save credit card URL with invalid format
```

Comment:
I don't usually write feature-level specs for "sad path" cases. Since feature specs are slower, filling a test suite with feature specs will inevitably lead to a super slow suite. We do want coverage for validations, but I think model-level specs should be enough.  What do you think?